rocksdb_admin/admin_handler.cpp

```diff
@@ -2182,4 +2183,18 @@ std::vector<std::string> AdminHandler::getAllDBNames() {
     return db_manager_->getAllDBNames();
 }
 
+void AdminHandler::initS3Tmp() {
```

Comment:
can we unit test this. 

We could also extract the delete-and-create logic into a separate function, if that makes  testing easier.

Reply:
@jaricftw how would background threads going to call an instance function on AdminHandler before it's constructed?

I grepped for s3_tmp this is the only place where this exists.

For thrift server initialization we would do the following
- Build the handler
- Build the server using the handler
- Server start accepting connections